packages/contracts/docs/developer-portal/04-framework-lifecycle/04-sub-processes/02-contract-initialization.md

```diff
@@ -0,0 +1,127 @@
+# Initialization Change
+
+```mermaid
+flowchart TD
+	init[["Change in initialization"]]
+
+	%% Checks
+	isImmutable{"is immutable \n variable?"}
+	isUpgradeable{"is contract UUPS \n upgradeable?"}
+	isCloneable{"is contract \n cloneable?"}
+    isVersioned{"is contract \n versioned?"}
```

Comment:
are all the versionable contracted are versioned now?
do we need to have a flow to version un-versioned contract?

Reply:
On develop, all contracts are versioned now. 
In the 1.3.0 release candiate, only `DAO`, `DAOFactory`, `PluginRepo` and `PluginRepoFactory` are versioned.